### PR TITLE
chore(RHTAPWATCH-1267): Update quay repository for project-controller

### PR DESCRIPTION
**Pull request by hmariset**
Update project-controller quay repository to change to konflux-ci organization from redhat-appstudio.

Jira-Url: https://issues.redhat.com/browse/RHTAPWATCH-1267

### CODE REVIEW

**Review by amisstea — APPROVED** [review]
/lgtm

**Review by ifireball — CHANGES_REQUESTED** [review]
Were the corresponding needed changes made to the project-controller's release pipeline? I haven't seen a PR doing that, nor am I seeing one linked to the Jira issue.

**Comment by hmariset** [comment]
@ifireball Can you give approval on this PR since the release pipeline changes are already merged.

**Review by ifireball — APPROVED** [review]

File: components/project-controller/development/kustomization.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ resources:
 
 images:
 - name: konflux-project-controller
-  newName: quay.io/redhat-appstudio/project-controller
+  newName: quay.io/konflux-ci/project-controller
   newTag: 6773b77f353cc019166754ecd4d563035968019c
```
Comment (ifireball) [inline components/project-controller/development/kustomization.yaml:10]: Need to ensure the tag actually matches a tag we have for this image in quay.io

**Review by ifireball — CHANGES_REQUESTED** [review]
it seems the PR as it is wouldn't work because the image tag does not exist in quay.io

**Pull request merged**